01-syntax/05-oop/tableDataFromBook.js

```diff
@@ -0,0 +1,39 @@
+/* global repeat TextCell  */
+
+function repeat(string, times) {
```

Comment:
Looks like `repeat` is a helper function, also I see that you reuse it in `StretchCell`.

I propose to assign it as a `static method` to `TextCell` like 

```js

// ...
TextCell.repeat = function repeat(){ 
    // ...
};

```

And then reuse in `StretchCell`